airbyte-integrations/connectors/source-netsuite/CHANGELOG.md

```diff
@@ -1,9 +0,0 @@
-# Changelog
-
```

Comment:
It's unlikely that we would actually maintain a changelog, so that thing is probably best removed. Thank you for the cleanup.